src/color.js

```diff
@@ -99,7 +99,7 @@ export default class Color {
 
 			// b) Remove instance properties from previous color space
 			for (let prop in this.space.instance) {
-				if (this.hasOwnProperty(prop)) {
+				if (Object.prototype.hasOwnProperty.call(prop)) {
```

Comment:
It's from eslint's [no-prototype-builtins](https://eslint.org/docs/rules/no-prototype-builtins). The idea is that on untrusted objects this may be trouble. Since this is our own object this is more of a personal choice to not use a disable-next-line